ara/ui/templates/partials/host_results.html

```diff
@@ -145,6 +145,13 @@
                     {% else %}
                       {% include "partials/result_status_icon.html" with status=result.status %}
                     {% endif %}
+                    {% if result.task.tags|length == 0 %}
```

Comment:
nitpick, would rather have the version with the space like the one you wrote below for consistency

```suggestion
                    {% if result.task.tags | length == 0 %}
```